client/src/components/ideas/CreateIdea.vue

```diff
@@ -19,7 +19,7 @@
 
         <div class="create__form-tags">
           <div class="create__tag-wrap">
-            <span class="create__form-tag" v-for="(tag, index) in tags">
+            <span class="create__form-tag" v-for="(tag, index) in tags" v-bind:key="index">
```

Comment:
Why this change?

Reply:
You need to add keys when iterating over elements.